src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,24 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
```

Comment:
Consider checking if the authorization header starts with 'Bearer ' to ensure the token is extracted correctly. This can prevent issues if the header format is incorrect.